genesis/model/object/context.go

```diff
@@ -14,30 +14,29 @@
  *    limitations under the License.
  */
 
-package resolver
+package object
```

Comment:
For now you need to show that it is resolver context.   Rename file to resolver.go and set there Resolver interface and all 3 implementation.